### PR TITLE
Inspector

### PR DESCRIPTION
**Pull request by xepozz**
I want to introduce some real-time application inspector that will be able to:
1. Dump any object from the container
2. Inspect DI config
3. Inspect params
4. Inspect shared envs
5. A lot of funny and useful things

Also it may be able to run some CLI inspections like `psalm`, `phpunit`, `deptrac` and others.

**Important thing is it won't know about any requests as the debugger know. It may inspect application and all related things.**

I made some example and want to discuss that idea.
Globally it may be separated from debug package, but as a PoC I made it with debug panels and API.

### CODE REVIEW

**Comment by xepozz** [comment]
Screenshot just for imagination
image

**Comment by xepozz** [comment]
Ability to run tests from ui. 
image

**Comment by xepozz** [comment]
Now it can read DI config
telegram-cloud-photo-size-2-5411548876613206844-y

And also params for DI config
image

File: src/Controller/InspectController.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    public function __construct(
+        private DataResponseFactoryInterface $responseFactory,
+    ) {
+    }
+
+    public function config(ContainerInterface $container): ResponseInterface
+    {
+        $config = $container->get(ConfigInterface::class);
+
+        // TODO: pass different envs
```
Comment (samdark) [inline src/Controller/InspectController.php:30]: ???

File: src/Controller/InspectController.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        $data = $config->get('web');
+        ksort($data);
+        foreach ($data as &$value) {
+//            $value = get_debug_type($value);
```
Comment (samdark) [inline src/Controller/InspectController.php:34]: ???

File: src/Controller/InspectController.php (hunk 1 above)
Comment (xepozz) [reply src/Controller/InspectController.php:30]: It's technically possible to fetch `console` group. But now it's unnecessary.

Comment (xepozz) [reply src/Controller/InspectController.php]: I've added it, but `$request['group']` doesn't exist now.

**Review by samdark — APPROVED** [review]

**Pull request merged**